tests/components/tessie/test_select.py

```diff
@@ -0,0 +1,65 @@
+"""Test the Tessie select platform."""
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant.components.select import (
+    DOMAIN as SELECT_DOMAIN,
+    SERVICE_SELECT_OPTION,
+)
+from homeassistant.components.tessie.const import TessieSeatHeaterOptions
+from homeassistant.const import ATTR_ENTITY_ID, ATTR_OPTION, STATE_OFF
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+
+from .common import ERROR_UNKNOWN, TEST_RESPONSE, setup_platform
+
+
+async def test_select(hass: HomeAssistant) -> None:
+    """Tests that the select entity is correct."""
+
+    assert len(hass.states.async_all(SELECT_DOMAIN)) == 0
+
+    await setup_platform(hass)
+
+    assert len(hass.states.async_all(SELECT_DOMAIN)) == 5
+
+    entity_id = "select.test_seat_heater_left"
+    assert hass.states.get(entity_id).state == STATE_OFF
+
+    # Test changing select
+    with patch(
+        "homeassistant.components.tessie.select.set_seat_heat",
```

Comment:
Please only patch out the library

Reply:
Do you mean `with patch("tessie_api.set_seat_heat",` because that does not work.